samples/java-shopping-cart-quickstart/src/main/proto/shopping/cart/kalix_policy.proto

```diff
@@ -1,14 +1,14 @@
 // This is the default Access Control List (ACL) for all components of this Kalix Service
 syntax = "proto3";
 
-package shopping.cart;
+package customer;
```

Comment:
Previous version maybe?
All packages seem to be set to `customer` now (thanks copy-paste I suppose)... Not critical but if we want to keep them aligned you might want to revisit them.

Reply:
yep, I was copying and pasting stuff without even looking at the package name. 

I will revert and start over again.